src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncController.scala

```diff
@@ -271,7 +285,8 @@ class FastSyncController(
       fastSyncStateStorage.purge()
       log.info("Fast sync finished")
       context.parent ! FastSyncDone
-      context stop self
+      context become idle
+      self ! StartRegularSync
```

Comment:
or maybe even better, move this lines to separate method (`startRegularSync` ?)

and then update `processSyncing` to something like:
```
if (fastDownloadFullySynced) {
  finishFastSync()
  startRegularSync()
```